PS8lite.Rmd

```diff
@@ -187,10 +188,44 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
+fit_control <- trainControl(method = "cv", number = 10)
 
+# Setup search grid of "tuning" parameters
+mtry <- 2:5
```

Comment:
Dunno if you tried increasing the "max number of predictors to consider at each node in the tree" from 5 to something higher, like 36 which is the number of predictors. But if you did, it would slow the run-time of the algorithm to a crawl. 

This is a great example of a problem where computational constraints become an issue! In other words, method A might yield "better predictions" than method B, but if method A takes a week to run and method B takes only a few minutes, you might sacrifice a little predictive power for speed!